Gemfile

```diff
@@ -4,29 +4,28 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.22'
-gem 'rack-cache', '< 1.3.0'
+gem 'rails', '4.1.5'
```

Comment:
I'd rather keep it pinned to a specific version, as there are sometimes API changes between patch-level version bumps in rails.